### PR TITLE
fix error of timeout when using Truffle to deploy contract

### LINKED ISSUE #767: the transaction keep pending until timeout when i using truffle deploy contract
__System info:__ [Include Ethermint commit, operating system name, and other relevant details]

__Steps to reproduce:__

1. [First Step]
run a local node
2. [Second Step]
install truffle to my system
3. [and so on...]
config truffle network to connect local node, run truffle deploy

__Expected behavior:__ [What you expected to happen]
deploy succeed

__Actual behavior:__ [What actually happened]
tx keep pending until timeout

__Additional info:__ [Include gist of relevant config, logs, etc.]

### PR DESCRIPTION
**Pull request by KamiD**
Closes: #767 

## Description

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

**Comment by araskachoi** [comment]
can you update the branch, looks good to merge once that's done!

**Review by araskachoi — DISMISSED** [review]

File: rpc/types/utils.go [hunk 1]
```diff
@@ -125,7 +125,7 @@ func EthTransactionsFromTendermint(clientCtx clientcontext.CLIContext, txs []tmt
 			continue
 		}
 		// TODO: Remove gas usage calculation if saving gasUsed per block
-		gasUsed.Add(gasUsed, ethTx.Fee())
+		gasUsed.Add(gasUsed, big.NewInt(int64(ethTx.GetGas())))
```
Comment (araskachoi) [inline rpc/types/utils.go:128]: GetGas() just gives the gas limit correct?
would it be better to use:
`gasUsed.Add(gasUsed, ethTx.Fee().Int64())`

Comment (KamiD) [reply rpc/types/utils.go:128]: I think here need return gas used, not gas fee used, also, the number of Fee() is too big to return to the interface of web3.js, that will be crashed in web3.js lib cause too large number transform

Comment (KamiD) [reply rpc/types/utils.go:128]: ethTx.Fee() return gaslimit * gasPrice, its much larger than real number of gas costed

**Review by araskachoi — APPROVED** [review]

**Pull request merged**